src/libs/auth.js

```diff
@@ -111,7 +111,7 @@ export const signIn = async (includeBillingScope = false) => {
   if (!isGoogleAuthority()) {
     authStore.update(state => ({ ...state, hasGcpBillingScopeThroughB2C: includeBillingScope }))
   }
-
+  Ajax().Metrics.captureEvent(Events.userLogin, { authProvider: user.profile.idp })
```

Comment:
Note, `signIn` is also called on the billing projects page to sign in with the billing scope. Thus, this may be producing more events than intended.

Reply:
Ah, thanks for pointing that out! I'll make an update to filter that out

Reply:
Am I correct in thinking that if includeBillingScope is false that means it's not coming from the billing projects page?

Reply:
SignInButton and TopBar call signIn with includeBillingScope set to false.

Reply:
Had further conversation with product, sounds like they only care about logins by unique user and users are tracked in mixpanel, so this shouldn't be a problem. Thanks for the heads up so we could make sure!